data/human/pirates/pirate intro.txt

```diff
@@ -0,0 +1,730 @@
+			choice
+				`	"What's going on here?"`
+				`	(Stroll into the middle of town. Lock eyes with the leader of the slavers. Assume a gunfighter's stance.)`
+					goto duel
+				`	(Return to your ship. This is none of your business.)`
+					goto retreat
+				`	(Return to your ship... and start strafing the slavers.)`
+					goto attack
+			`	"No business of yours, friend. We know who you are, and we thank you for running off the last band of lowlifes that laid claim to these bodies. Now, if you'll excuse, we've got work to do. Piss off." The slaver chief roundly ignores any further attempts to communicate, but he at least seems to be in a good enough mood to not have you shot on the spot.`
+			choice
+				`	"These people are my responsibility, and I'll fight you for their freedom."`
+				`	(Return to your ship. This is none of your business.)`
+					goto retreat
+				`	(Return to your ship... and start strafing the slavers.)`
+					goto attack	
+			label duel
+			`	The leader of the slavers turns to face you, attention finally pulled from his task. "You've got some nerve picking a fight with us, kid. You really think these poor bastards are worth dying for?"`
+			`	Noon's sun beats down on you. Even with the chill that usually haunts this region of the planet at this time of year, it's enough to bring sweat to your brow. Silence fills the town square, and slavers shy back to leave only four things that matter in the area: You, the slaver boss, and your respective pistols.`
+			`	A bird cries in the distance.`
+			choice
+				`	(Draw!)`
+			`	The slaver boss is fast, but you're faster. You're able to slip your sidearm out of its holster, level it, and cleanly blow him away before he can so much as draw a bead on you - and so it's with the thrill of victory that you die, as you're unceremoniously cut down by massed fire from the gang of slavers surrounding your little duel.`
+			`	The few colonists that are left behind after the slavers have filled their holds honor you with a short burial and a bottle of local moonshine poured out over your cairn in respect for the brave, but fairly ill-conceived, attempt to defend them.`
+				die
+			label attack
+			`	You retreat to your ship, unable to help these poor souls...`
+			`	... Or at least, that's what you want the slavers to think. They're caught unaware as you streak back towards the town, weapons hot and firing on clusters of slavers caught out in the open. Unarmored gangsters are no match for starship-grade weaponry, and your first volley reduces a good number of the guards to so much greasy mist. Your second pass initiates a full rout, as the slavers beat a hasty retreat from the village back to the landing pad.`
+			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`
+				launch
```

Comment:
In case the player is unarmed.

```suggestion
			`	Survivors pour through the hatches and into their ships. Engines are warmed, weapons are brought online, and soon enough you're roaring out of atmosphere to finish this fight in the skies above.`
				to display
					"armament deterrence" > 0
				launch
			`	Survivors pour through the hatches and into their ships. You have no choice but to retreat in your unarmed ship, hoping you can outrun them before they shoot you down.`
				launch
```